resources/language/English/strings.po

```diff
@@ -180,3 +180,7 @@ msgstr ""
 msgctxt "#30048"
 msgid "WEEK {0}"
 msgstr ""
+
+msgctxt "#30049"
+msgid "Use InputStream Adaptive"
```

Comment:
I think that name would be confusing as adaptive bitrate is not yet activated within inputstream.adaptive. Right now, the default value is measured by measuring the download speed of the segment.